### PR TITLE
Feature/60603 fix a11y issues

### PR DESCRIPTION
**Pull request by sushmi21**
Please test this PR with the PR in webchat widget repo https://github.com/Cognigy/WebchatWidget/pull/405 

EDIT: Please not that the adaptive card elements focus styles are not yet refined in this PR

### CODE REVIEW

**Comment by fabclj** [comment]
Screenshot from 2024-04-05 14-48-09

The button named "Call Cognigy" appears only on hover or focus.

**Comment by fabclj** [comment]
On adaptive cards, focus styles are a bit different than the rest of components>
You can see on this endpoint:
https://endpoint-dev.cognigy.ai/18ed9261b60ae4f3945bf8f8ebc047fc62ff74bb4996837e00fa6a761a164106

**Comment by sushmi21** [comment]
> On adaptive cards, focus styles are a bit different than the rest of components You can see on this endpoint: https://endpoint-dev.cognigy.ai/18ed9261b60ae4f3945bf8f8ebc047fc62ff74bb4996837e00fa6a761a164106

Yes, I wanted mention in PR description. I was unable to bring the same behaviour for adaptive cards. I have to ask Dmitrii.

**Comment by sushmi21** [comment]
> Screenshot from 2024-04-05 14-48-09
> 
> The button named "Call Cognigy" appears only on hover or focus.
> 
> Weird! I will check if this works in main first

I checked and it is an existing bug in main. I will investigate and fix it in this branch

File: src/common/Buttons/PrimaryButton.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ interface PrimaryButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
 	onEmitAnalytics?: ActionButtonsProps["onEmitAnalytics"];
 }
 
-const PrimaryButton: FC<PrimaryButtonProps> = props => {
+const PrimaryButton = forwardRef<HTMLButtonElement, PrimaryButtonProps>((props, ref) => {
```
Comment (fabclj) [inline src/common/Buttons/PrimaryButton.tsx:19]: do we maybe need the same logic on SecondaryButton component?

Comment (sushmi21) [reply src/common/Buttons/PrimaryButton.tsx:19]: Right now it is not necessary, but I can add that.

Comment (fabclj) [reply src/common/Buttons/PrimaryButton.tsx:19]: I would suggest to add that in this PR, otherwise later we can easily miss that detail.

**Comment by sushmi21** [comment]
> > On adaptive cards, focus styles are a bit different than the rest of components You can see on this endpoint: https://endpoint-dev.cognigy.ai/18ed9261b60ae4f3945bf8f8ebc047fc62ff74bb4996837e00fa6a761a164106
> 
> Yes, I wanted mention in PR description. I was unable to bring the same behaviour for adaptive cards. I have to ask Dmitrii.

I will handle this in a new PR

**Review by fabclj — APPROVED** [review]

**Pull request merged**